generator/parser/parser.cpp

```diff
@@ -2300,12 +2300,12 @@ bool Parser::parseBaseClause(BaseClauseAST *&node)
   return true;
 }
 
-bool Parser::parseInitializer(InitializerAST *&node)
+bool Parser::parseInitializer(InitializerAST *&node, bool allowNewStyle)
 {
   std::size_t start = token_stream.cursor();
 
   int tk = token_stream.lookAhead();
-  if (tk != '=' && tk != '(')
+  if (tk != '=' && tk != '(' && !(allowNewStyle && tk == '{'))
```

Comment:
I think `&& (!allowNewStyle || tk == '{')` would be better readable, but that may be just me.

Reply:
I thought about that, but found that for me the condition is better readable that way.
By the way, it would be `&& (!allowNewStyle || tk != '{')`

Reply:
True of course 😀 And as I wrote, a matter of taste...